### PR TITLE
Python 3.10

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

- Upgrade to Python 3.10
- Also started using gevent instead of eventlet.

Tested by running:
- the `ci/Dockerfile` image from this PR
-  the api devcontainer
Send messages with both attachments and file links.

# Test instructions | Instructions pour tester la modification

Run this branch together with any api and send attachments.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -15,7 +15,8 @@ eventlet==0.30.2
 
 awscli-cwlogs==1.4.6
 
-git+https://github.com/cds-snc/notifier-utils.git@48.0.0#egg=notifications-utils
+# git+https://github.com/cds-snc/notifier-utils.git@48.0.0#egg=notifications-utils
+git+https://github.com/cds-snc/notifier-utils.git@python-3.10#egg=notifications-utils
```
Comment (sastels) [inline requirements.txt:19]: change when we merge the utils PR

**Comment by sastels** [comment]
Checked that the `ci/Dockerfile` image builds and runs locally.

**Comment by sastels** [comment]
Ran `ci/Dockerfile` locally with api devcontainer and sent attachements / links.

**Review by andrewleith — APPROVED** [review]
LGTM - was able to run this locally and view linked files.

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -33,4 +33,4 @@ RUN pip install --upgrade pip
 
 COPY .devcontainer/scripts/notify-dev-entrypoint.sh /usr/local/bin/
 
-EXPOSE 6012
+EXPOSE 7000
```
[the remark this answers is not in the document]
Comment (sastels) [reply .devcontainer/Dockerfile:36]: yeah I assume when Pat made the devcontainer for this repo he just copied api's but I'm bit confused as to what was happening before...

**Pull request merged**